package.json

```diff
@@ -29,7 +29,7 @@
     "ethereumjs-abi": "^0.6.8",
     "ethereumjs-tx": "^2.1.0",
     "ethereumjs-util": "^6.1.0",
-    "ganache-cli": "^6.7.0",
+    "ganache-cli": "^6.8.0-istanbul.0",
     "mathjs": "^4.3.0",
```

Comment:
this version default version is without Istanbul?

Reply:
Yup. Runs on Petersburg by default.